pliers/transformers/base.py

```diff
@@ -235,11 +235,33 @@ def _iterate(self, stims, validation='strict', *args, **kwargs):
         batches = batch_iterable(stims, self._batch_size)
         results = []
         for batch in progress_bar_wrapper(batches):
-            res = self._transform(batch, *args, **kwargs)
+            use_cache = config.get_option('cache_transformers')
```

Comment:
Logic all looks good to me, but maybe add a comment or two in this section clarifying what's going on, for posterity.